ocaml/idl/datamodel_vm.ml

```diff
@@ -119,6 +125,16 @@ let set_actions_after_crash =
       ]
     ~allowed_roles:_R_VM_ADMIN ()
 
+let set_actions_after_softreboot =
+  call ~name:"set_actions_after_softreboot" ~in_oss_since:None
+    ~in_product_since:rel_rio ~doc:"Sets the after_softreboot parameter"
```

Comment:
version here is wrong, this is a new field, hasn't been there since Rio.

Reply:
Remove `in_product_since:rel_rio` and add `lifecycle:[]` to add it as a new prototype.